src/Uno.UI/UI/Xaml/Media/Animation/Timeline.animation.netstdref.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Windows.UI.Xaml.Media.Animation
+{
+	partial class Timeline
+	{
+		partial class AnimationImplementation<T>
```

Comment:
Codacy found an issue: ['T' is not used in the class.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5872302)